WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/BlogDetailHeaderView.swift

```diff
@@ -53,16 +55,11 @@ class BlogDetailHeaderView: UIView {
     }
 
     @objc func refreshIconImage() {
-        if let blog = blog,
-            blog.hasIcon == true {
-            titleView.siteIconView.imageView.downloadSiteIcon(for: blog)
-        } else if let blog = blog,
-            blog.isWPForTeams() {
-            titleView.siteIconView.imageView.tintColor = UIColor.listIcon
-            titleView.siteIconView.imageView.image = UIImage.gridicon(.p2)
-        } else {
-            titleView.siteIconView.imageView.image = UIImage.siteIconPlaceholder
-        }
+        guard let blog else { return }
```

Comment:
What do you think keeping the original logic where using `UIImage.siteIconPlaceholder` if `blog` is nil?

Reply:
I wonder what circumstances would cause `blog` to be nil? 

That seems very odd

Reply:
From a quick look at related code, I don't see any place that would set it to nil. It's probably an API design thing, where it's made as an optional but don't actually need to be one.

Reply:
If there is no blog, this header isn't shown, so I think we should be good.